client/blocks/site-address-changer/style.scss

```diff
@@ -5,14 +5,13 @@
 	padding: 48px 48px 0;
 	max-width: 800px;
 
-}
-
-.dialog__action-buttons {
-	border: 0;
-	padding: 35px 48px 48px;
+	.dialog__action-buttons {
+		border: 0;
+		padding: 35px 48px 48px;
```

Comment:
@Aurorum Looks like we'll break the style of `<SiteAddressChanger>` with this change. Here's how you'd test the site address changer: https://wordpress.com/support/changing-site-address/#change-your-word-press-com-site-address

|before|after|
|-------|-----------------|
|<img width="848" alt="Screen Shot 2024-11-04 at 8 51 24 AM" src="https://github.com/user-attachments/assets/f5abb5a7-b06e-4378-9fb8-4b2cef0a39e3">|<img width="858" alt="Screen Shot 2024-11-04 at 8 51 19 AM" src="https://github.com/user-attachments/assets/c168faa1-a8a0-43c6-a2be-b0a2f7936468">|

To we can customize the button style for this dialog box, we may consider:

* Wrapping the `<Dialog>` component for the `<SiteAddressChanger> [here](https://github.com/Automattic/wp-calypso/blob/d011f3b3dd669d52a300b3923faced62280f0154/client/blocks/site-address-changer/index.jsx#L554) with a div, assign a custom class name to it.

* Updating `<Dialog>` to apply the custom class name to the [<ButtonBar>](https://github.com/Automattic/wp-calypso/blob/d011f3b3dd669d52a300b3923faced62280f0154/packages/components/src/dialog/index.tsx#L100).

Reply:
Ah, is the screenshot you shared the outcome of removing `className="site-address-changer"`? If so, I think it's the right call to to stick with the base style from `<Dialog>`!

Reply:
Ah, perfect - yep, the screenshot is with the class name removed and just the default Dialog styles. :)